core/include/keyman/keyman_core_api_context.h

```diff
@@ -96,7 +83,7 @@ km_core_state_get_intermediate_context(km_core_state *state, km_core_context_ite
 ### `km_core_context_items_dispose`
 ##### Description:
 Free the allocated memory belonging to a `km_core_context_item` array previously
-returned by `km_core_context_items_from_utf16` or `km_core_context_get`
+returned by `km_core_context_items_from_utf16` or `context_get`
```

Comment:
```suggestion
returned by `context_items_from_utf16` or `context_get`
```

Reply:
Hmm, if both methods that will return a `km_core_context_item` array are internal now, I wonder if `km_core_context_items_dispose` still has to  be public?

Reply:
`km_core_state_get_intermediate_context` also uses `km_core_context_item`.